.github/workflows/e2e.yml

```diff
@@ -24,28 +24,28 @@ jobs:
         uses: actions/setup-go@v1
         with:
           go-version: 1.19.x
-      - name: Setup GOPATH and AVAPATH
+      - name: Setup GOPATH and CAMPATH
         run : |
-          mkdir -p $GOPATH/$AVAPATH
+          mkdir -p $GOPATH/$CAMPATH
       - name: Install autoremove
         run: sudo apt-get autoremove -y
-      - name: Clone and Build AvalancheGo
+      - name: Clone and Build Camino Node
         run : |
-          cd $GOPATH/$AVAPATH
+          cd $GOPATH/$CAMPATH
           git clone https://github.com/chain4travel/camino-node
           cd camino-node
           ./scripts/build.sh
-      - name: Checkout ANR
+      - name: Checkout Camino Network Runner
         run : |
-          cd $GOPATH/$AVAPATH
+          cd $GOPATH/$CAMPATH
           git clone https://github.com/chain4travel/camino-network-runner
-      - name: Starting ANR
+      - name: Starting Camino Network Runner
         run: |
-          cd $GOPATH/$AVAPATH
+          cd $GOPATH/$CAMPATH
           cd camino-network-runner
           git checkout v0.1.4
           go run ./examples/local/fivenodenetwork/main.go &
-      - name: AvalancheJS E2E Test
+      - name: CaminoJS E2E Test
         env:
           AVALANCHEGO_IP: localhost
```

Comment:
**AVALANCHEGO_IP** and **AVALANCHEGO_PORT** should be changed to **CAMINOGO_IP** and **CAMINO_PORT**